pkg/maps/lbmap/lbmap.go

```diff
@@ -505,12 +531,25 @@ func (*LBBPFMap) DumpServiceMaps() ([]*loadbalancer.SVC, []error) {
 		if err != nil {
 			errors = append(errors, err)
 		}
+		err = RevNat6Map.DumpWithCallback(parseRevNatEntries)
+		if err != nil {
+			errors = append(errors, err)
+		}
 		err = Service6MapV2.DumpWithCallback(parseSVCEntries)
 		if err != nil {
 			errors = append(errors, err)
 		}
 	}
 
+	for _, svcKey := range inconsistentServiceKeys {
```

Comment:
`DumpServiceMaps` as the name implies should be just dumping them and not deleting entries. Please move this into a separate method. Other than that this seems like a reasonable approach to me!